.pre-commit-config.yaml

```diff
@@ -1,3 +1,4 @@
+repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks.git
     sha: v0.7.1
     hooks:
```

Comment:
you can drop all the language_version in this file now that we're on a new-enough pre-commit

Reply:
Neat